gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -133,6 +139,25 @@ public AddSpecResponse onAddSpec(Spec addedSpec) {
     return new AddSpecResponse<>(null);
   }
```

Comment:
Just an FYI, this also gets called during updating a flow. But since we have a condition of checking the flow is scheduled or not and we don't expect users to update an adhoc flow, we should be fine.

Reply:
Thanks for the callout, will add it for the test suite